extensions/resteasy/deployment/src/test/java/io/quarkus/resteasy/test/NotFoundExceptionMapperTestCase.java

```diff
@@ -54,15 +54,19 @@ public void testJsonResourceNotFound() {
     }
 
     @Test
-    public void shouldDisplayNewAddedFileIn404ErrorPage() {
+    public void shouldDisplayNewAddedFileIn404ErrorPage() throws InterruptedException {
         String CONTENT = "html content";
         test.addResourceFile(META_INF_RESOURCES + "index2.html", CONTENT);
 
+        Thread.sleep(700);
```

Comment:
wouldn't it be better to use Awaitibility and poll until it passes?

Reply:
Await on the `RestAssured` call passing?

And then have a max time of 2 seconds?

Is that preferred? I can adjust

Reply:
We currently don't have Awaitility as a dependency, is it ok to add?

Reply:
I have extensions using it I believe.

Reply:
ugh, I was looking at the wrong BOM!

Reply:
Tried switching to awaitility, and it has a failure with one, not entirely sure why

Reply:
Awaitility seems to lead to just as much raciness unfortunately